server/requirements.txt

```diff
@@ -8,3 +8,4 @@ gunicorn
 humanize
 pyesbulk>=2.0.1
 requests
+sqlalchemy
```

Comment:
Need to add `sqlalchemy_utils` to pick up the database bootstrapping dependency in `shell.py` ...